L1Trigger/L1TCaloLayer1/src/L1TCaloLayer1FetchLUTs.cc

```diff
@@ -40,6 +41,7 @@ bool L1TCaloLayer1FetchLUTs(
     bool useECALLUT,
     bool useHCALLUT,
     bool useHFLUT,
+    bool useHCALFBLUT,
```

Comment:
I miss where this `bool` is used here: maybe you forgot to implement a check for it?

Reply:
Thanks for noticing this error. I've added the missing check for `useHCALFBLUT` in the latest commit.